R/length-params.R

```diff
@@ -0,0 +1,158 @@
+#' Download metadata catch types
+#'
+#' This function download airtable tables containing metadata of catch types.
+#'
+#' @param log_threshold The (standard Apache logj4) log level used as a
+#'   threshold for the logging infrastructure. See [logger::log_levels] for more
+#'   details
+#'
+#' @return
+#' @export
+#'
+get_catch_types <- function(log_threshold = logger::DEBUG){
+
+  pars <- read_config()
+
+  metadata <- get_preprocessed_metadata(pars)
+
+  catch_types <- metadata$catch_types
+
+  fao_records <- air_get_records(table="fao_catch",
+                                 base_id=pars$metadata$airtable$base_id,
+                                 api_key=readLines("auth/airtable-key")) %>%
+    air_records_to_tibble()
```

Comment:
I think it would be best if we add this to the ingest metadata tables step so we get both tables at once from the metadata and we can ensure that we're using the same version of the metadata throughout a run of the pipeline.